### PR TITLE
small edits

### PR DESCRIPTION
**Pull request by sascala**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,19 +142,19 @@ Most services *can* be installed on the Medium cluster, but not all at the same
...
 
-    To avoid entering your password on `vagrant up` & `vagrant destroy` you may enable [passwordless sudo](https://github.com/smdahlen/vagrant-hostmanager#passwordless-sudo).
+    To avoid entering your password on `vagrant up` & `vagrant destroy`, enable [passwordless sudo](https://github.com/smdahlen/vagrant-hostmanager#passwordless-sudo).
 
     On some versions of Mac OS X, installing vagrant plugins may require [installing a modern version of Ruby](./docs/install-ruby.md).
 
 1. Download the DC/OS Installer
 
-    If you don't already have a DC/OS Installer downloaded, you'll need to select and download one of the [supported versions](#supported-dcos-versions).
+    If you don't already have a DC/OS installer downloaded, you'll need to select and download one of the [supported versions](#supported-dcos-versions).
```
Comment (karlkfi) [inline README.md]: "DC/OS Installer" isn't a proper name?

File: README.md [hunk 2]
```diff
@@ -146,19 +142,19 @@ Most services *can* be installed on the Medium cluster, but not all at the same
...
 
-    Once downloaded, move `dcos_generate_config.sh` to the root of the repo (the repo will be mounted into the vagrant machines as `/vagrant`).
+    Once downloaded, move the installer, `dcos_generate_config.sh`, to the root of the repo (the repo will be mounted into the vagrant machines as `/vagrant`).
```
Comment (karlkfi) [inline README.md]: maybe (`dcos_generate_config.sh`)?

File: README.md [hunk 3]
```diff
@@ -258,14 +254,27 @@ When selecting which machines to deploy, the following constraints must be obser
...
 
 **IMPORTANT**: Make sure your local machine has enough memory to launch all your desired VMs, otherwise your machine may lock up as all the memory is consumed.
 
+# Destroy Your Cluster
+
+The following command destroys your cluster and any data stored there:
+
+`vagrant destroy -f`
+
+# Upgrade dcos-vagrant
```
Comment (karlkfi) [inline README.md]: "Upgrade DC/OS Vagrant"

File: README.md (hunk 1 above)
Comment (sascala) [inline README.md]: Doesn't look it, eg: https://docs.mesosphere.com/advanced/

File: README.md (hunk 2 above)
Comment (sascala) [inline README.md]: sure

File: README.md (hunk 3 above)
Comment (sascala) [inline README.md]: You seem to use both.

Comment (karlkfi) [inline README.md]: One's the product, ones the repo.

Comment (sascala) [inline README.md]: I see. So in this sentence, "To upgrade to a new version of docs-vagrant," you're talking about the repo?

File: README.md (hunk 1 above)
Comment (karlkfi) [inline README.md]: bleh. fine. they need to rename the whole thing already. no one knows what to call the dcos_generate_config.sh

File: README.md (hunk 3 above)
Comment (karlkfi) [inline README.md]: That should change too, i guess...

Comment (sascala) [inline README.md]: :-)

**Pull request merged**